src/main/java/subway/domain/StationLine.java

```diff
@@ -53,18 +47,20 @@ public StationLine(String name, String color, Station upStation, Station downSta
         section.apply(this);
     }
 
+    public List<StationLineSection> getSections() {
+        return sections.getSections();
+    }
```

Comment:
피드백 반영 👍 
방어적 복사를 위해 불변 컬렉션으로 리턴하면 어떨까요?
```suggestion
    public List<StationLineSection> getSections() {
        return Collections.unmodifiableCollection(sections.getSections());
    }
```

Reply:
불변컬랙션으로 리턴시켜주는게 훨씬안전하겠네요 의견감사합니다 ㅎㅎ